hash/apple2_flop_clcracked.xml

```diff
@@ -49584,4 +48700,1980 @@ license:CC0-1.0
+		<info name="usage" value="Runs on any Apple ][ or later with 48K." />
+		<info name="version" value="B.00, 1981-01-03" />
+		<sharedfeat name="compatibility" value="A2,A2P,A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2021-10-08-->
+		<!--productivity program-->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="pfs vb.00 (4am crack).dsk" size="143360" crc="8106efe8" sha1="5f33707a6014b6d68c147130bcceb8ffb97240a7"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pfse841126">
+		<description>pfs: etat (French, version 1984-11-26) (4am crack)</description>
+		<!--French version of pfs: report-->
+		<year>1984</year>
+		<publisher>Software Publishing Corporation</publisher>
+		<info name="developer" value="Software Publishing Corporation" />
+		<info name="usage" value="Runs on any Apple ][+ or later with 64K." />
+		<info name="version" value="1984-11-26" />
+		<sharedfeat name="compatibility" value="A2P,A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2022-06-30-->
+		<!--productivity program-->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="pfs etat 1984-11-26 (4am crack).dsk" size="143360" crc="47aab309" sha1="60457eefaa21a29bc94b49e9f656aed2edd08ae6"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pfsfich841126">
+		<description>pfs: fichier (French, version 1984-11-26) (4am crack)</description>
+		<!--French version of pfs: file-->
+		<year>1984</year>
+		<publisher>Software Publishing Corporation</publisher>
+		<info name="developer" value="Software Publishing Corporation" />
+		<info name="usage" value="Runs on any Apple //e or later with 128K." />
+		<info name="version" value="1984-11-26" />
+		<sharedfeat name="compatibility" value="A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2022-07-10-->
+		<!--productivity program-->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="pfs fichier 1984-11-26 (4am crack).dsk" size="143360" crc="7bd22192" sha1="5831295341c97ac7007d047bbad87eb73ec18da5"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pfsfa00">
+		<description>pfs: file (version A.00) (4am crack)</description>
+		<year>1983</year>
+		<publisher>Software Publishing Corporation</publisher>
+		<info name="developer" value="Software Publishing Corporation" />
+		<info name="usage" value="Runs on any Apple //e or later with 64K." />
+		<info name="version" value="A.00" />
+		<sharedfeat name="compatibility" value="A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2021-11-11-->
+		<!--productivity program-->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="pfs file va.00 (4am crack).dsk" size="143360" crc="49980399" sha1="14451182f9d319d2c08a7f7fd68aec10654299b1"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pfsfb01">
+		<description>pfs: file (version B.01) (4am crack)</description>
```

Comment:
Similarly, are the pfs: file versions different revisions of the same software?  Is pfs: fichier the French version of pfs: file?